.build/build-resolver.xml

```diff
@@ -58,8 +58,11 @@
         <resolver:remoterepos id="all">
             <remoterepo id="resolver-central" url="${artifact.remoteRepository.central}"/>
             <remoterepo id="resolver-apache" url="${artifact.remoteRepository.apache}"/>
-            <!-- Snapshots are not allowed, but for feature branches they may be needed, so uncomment the below to allow snapshots to work -->
-            <remoterepo id="resolver-apache-snapshot" url="https://repository.apache.org/content/repositories/snapshots" releases="false" snapshots="true" updates="always" checksums="fail" />
+            <!-- Snapshot artifacts must not exist in nor be downloaded by any Cassandra release artifact.
+                 Please validate that all artifacts included in parent-pom-template.xml are release
+                 artifacts before committing.
+             -->
```

Comment:
```suggestion
            <!-- Snapshot artifacts must not exist in nor be downloaded by any Cassandra (source or binary) release artifacts.
                 
                 Please validate that all artifacts included in parent-pom-template.xml are release
                 artifacts before committing.
                 
                 Please ensure the next line is commented when cutting a release.
             -->
```

Reply:
Pushed a change that adds more context, instruction, and also comments this line out by default. I don't think we should resolve snapshot libraries by default in the repo and contributors should need to opt-in.